src/components/alphabet/main-space/main-space.js

```diff
@@ -0,0 +1,16 @@
+export default {
+    name: 'main-space',
```

Comment:
Please read https://vuejs.org/v2/style-guide/ please use naming convention from this style-guide